tests/unit/core/test_tracking.js

```diff
@@ -150,17 +168,89 @@ describe('Tracking', () => {
   });
 });
 
-describe('getAction', () => {
+describe('getAddonTypeForAction', () => {
   it('returns addon for TYPE_EXTENSION', () => {
-    expect(getAction(ADDON_TYPE_EXTENSION)).toEqual(TRACKING_TYPE_EXTENSION);
+    expect(getAddonTypeForAction(ADDON_TYPE_EXTENSION)).toEqual(
+      TRACKING_TYPE_EXTENSION,
+    );
+  });
+
+  it('returns addon for TYPE_OPENSEARCH', () => {
+    expect(getAddonTypeForAction(ADDON_TYPE_OPENSEARCH)).toEqual(
+      TRACKING_TYPE_EXTENSION,
+    );
   });
 
   it('returns theme for TYPE_THEME', () => {
-    expect(getAction(ADDON_TYPE_THEME)).toEqual(TRACKING_TYPE_THEME);
+    expect(getAddonTypeForAction(ADDON_TYPE_THEME)).toEqual(
+      TRACKING_TYPE_THEME,
+    );
+  });
+
+  it('returns addon:statictheme for TYPE_STATIC_THEME', () => {
+    expect(getAddonTypeForAction(ADDON_TYPE_STATIC_THEME)).toEqual(
+      TRACKING_TYPE_STATIC_THEME,
+    );
   });
 
   it('returns invalid for unknown type', () => {
-    expect(getAction('whatever')).toEqual('invalid');
+    expect(getAddonTypeForAction('whatever')).toEqual(TRACKING_TYPE_INVALID);
+  });
+});
+
+describe('getAddonEventCategory', () => {
+  it('returns Addon Installs Started category if type is a extension and install started is the action type', () => {
```

Comment:
Thanks for adding these tests @rebmullin. The descriptions are quite long, and I don't think you need to specify the actual value you expect in the description. It could just be something like `'returns the expected category when type is extension and installAction is install started'`. The same goes for the other tests below.